test/MotionPairMatch.test.ts

```diff
@@ -0,0 +1,13 @@
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
+
+import {MotionPairMatch} from '../src/Motions/PairMatch';
+
+export function run() {
+    // Test opening match
+    test("MotionPairMatch.matchOpening", () => {
+         var motion = MotionPairMatch.matchOpening({character : "\""});
+         //don't know how to mockup Position neither window.activeTextEditor
+    });
+}
+
```

Comment:
I'm so lazy that I haven't write any tests :disappointed: .
I think we'd choose another time to deal with all module test. Delete this file would be fine for now.